mockgcp/fixup-third-party.sh

```diff
@@ -55,9 +57,15 @@ find . -type f -print0 | xargs -0 sed -i -e "s@google\.pubsub@mockgcp.pubsub@g"
 find . -type f -print0 | xargs -0 sed -i -e "s@google/monitoring/@mockgcp/monitoring/@g"
 find . -type f -print0 | xargs -0 sed -i -e "s@google\.monitoring@mockgcp.monitoring@g"
 
+find . -type f -print0 | xargs -0 sed -i -e "s@google/spanner/@mockgcp/spanner/@g"
+find . -type f -print0 | xargs -0 sed -i -e "s@google\.spanner@mockgcp.spanner@g"
+
 find . -type f -print0 | xargs -0 sed -i -e "s@google/storage/@mockgcp/storage/@g"
 find . -type f -print0 | xargs -0 sed -i -e "s@google\.storage@mockgcp.storage@g"
 
+find . -type f -print0 | xargs -0 sed -i -e "s@google/spanner/@mockgcp/spanner/@g"
```

Comment:
NIt: this is duplicated with a few lines above (unless I'm missing something), but ... harmless.

Reply:
fixed